pydicom/multival.py

```diff
@@ -38,39 +39,44 @@ class or type corresponding to the VR.
         """
         from pydicom.valuerep import DSfloat, DSdecimal, IS
 
-        self.type_constructor = type_constructor
+        def list_type_constructor(x):
+            return type_constructor(x) if x != '' else x
```

Comment:
Should this not be `self.type_constructor(x)` since `typeconstructor` is set to `list_type_constructor` for DS etc?